framework/src/wifi_manager/wifi_manager_api.c

```diff
@@ -40,8 +40,8 @@
 			config->passphrase_length > WIFIMGR_PASSPHRASE_LEN ||		\
 			strlen(config->ssid) > WIFIMGR_SSID_LEN ||					\
 			strlen(config->passphrase) > WIFIMGR_PASSPHRASE_LEN) {		\
-			NET_LOGE(TAG, "AP configuration fails: too long ssid or passphrase %d\n", config->ssid_length);	\
-			NET_LOGE(TAG, "Make sure that length of SSID < 33 and length of passphrase < 65 %d\n", config->passphrase_length); \
+			NET_LOGE(TAG, "AP configuration fails: too long ssid or passphrase\n");	\
```

Comment:
In my opinion, this will make debug more hard, so If I were you I would divide if state.
```
if (config->ssid_length > WIFIMGR_SSID_LEN) {
   NET_LOGE("too long ssid\n");
   return WIFI_MANAGER_INVALID_ARGS;
}
if (config->passphrase_length > WIFIMGR_PASSPHRASE_LEN) {
   NET_LOGE("too long passphrase\n");
}
```

Reply:
Thanks for comment. As you said it's more clear to divide error condition. I'll modify it.